File: app/src/main/java/email/schaal/ocreader/api/json/v2/SyncResponse.java
package email.schaal.ocreader.api.json.v2;

import java.util.List;

import email.schaal.ocreader.database.model.Feed;
import email.schaal.ocreader.database.model.Folder;
import email.schaal.ocreader.database.model.Item;

/**
 * API response for sync call
 */
public class SyncResponse {
    private List<Folder> folders;
    private List<Feed> feeds;
    private List<Item> items;

    public List<Folder> getFolders() {
        return folders;
    }

    public void setFolders(List<Folder> folders) {
        this.folders = folders;
    }

    public List<Feed> getFeeds() {
        return feeds;
    }

    public void setFeeds(List<Feed> feeds) {
        this.feeds = feeds;
    }

    public List<Item> getItems() {
        return items;
    }

    public void setItems(List<Item> items) {
        this.items = items;
    }
}

File: app/src/main/java/email/schaal/ocreader/api/json/v2/SyncResponse.kt
package email.schaal.ocreader.api.json.v2

import email.schaal.ocreader.database.model.Feed
import email.schaal.ocreader.database.model.Folder
import email.schaal.ocreader.database.model.Item

/**
 * API response for sync call
 */
class SyncResponse {
    var folders: List<Folder>? = null
    var feeds: List<Feed>? = null
    var items: List<Item>? = null

}